multi_import/formats.py

```diff
@@ -1,15 +1,12 @@
+from csv import Error as NullError
+from io import BytesIO, StringIO
+
 import chardet
-import six
-from django.utils.translation import ugettext_lazy as _
-from tablib.compat import BytesIO, StringIO
+from django.utils.translation import gettext_lazy as _
 from tablib.core import Dataset, InvalidDimensions, UnsupportedFormat
 from tablib.formats import _csv, _json, _xls, _xlsx, _yaml
-from csv import Error as NullError
-
-try:
-    from yaml import CSafeDumper
-except (ModuleNotFoundError, ImportError):
-    CSafeDumper = None
```

Comment:
I forget the error code, but there was an error coming up so I enforced the import and added the dependency to pyproject.toml

Reply:
Hmm that module I think is only usable if pyyaml is built with libyaml: https://pyyaml.org/wiki/PyYAMLDocumentation. So I think the conditional import makes sense to keep since libyaml isn't required.

Reply:
Reverted the change here and remove the pyyaml requirement - https://github.com/sdelements/django-multi-import/pull/40/commits/6c8971ef4ebc86100a426848f42d9ae36d1b5ec5